lib/formatters/tapFormatter.mjs

```diff
@@ -37,7 +39,7 @@ export default function tapFormatter(results) {
 
```

Comment:
todo

The Names chapter was removed in #6390.

> [made up of lowercase words separated by hyphens](https://github.com/stylelint/stylelint/pull/6390/files#diff-7a62ae7246fc278776a7129004bcc382e494619fcda082e35c94b8a858122680)

@jeddy3 we need to put this back in the documentation.
It has the advantage of discarding `linebreaks`, `indentation` and the `report*` options as valid rules.